Tests/UI/BillingForm/Validator/AddressLine2ValidatorTests.swift

```diff
@@ -6,14 +6,14 @@ class AddressLine2ValidatorTests: XCTestCase {
     func testValidationWhenTextIsEmptyThenShouldBeTrue() {
         let expectedType = BillingFormCell.addressLine2(nil)
         let text = ""
-        let isValid = expectedType.validator.validate(text: text)
-        XCTAssertTrue(isValid)
+        let isInvalid = expectedType.validator.isInvalid(value: text)
+        XCTAssertTrue(isInvalid)
     }
 
     func testValidationWhenTextIsNonEmptyThenShouldBeFalse() {
         let expectedType = BillingFormCell.addressLine2(nil)
         let text = "addressLine2"
-        let isValid = expectedType.validator.validate(text: text)
-        XCTAssertFalse(isValid)
+        let isInvalid = expectedType.validator.isInvalid(value: text)
+        XCTAssertFalse(isInvalid)
```

Comment:
This is an example of why I was saying your naming of `isInvalid` should not be negative. We have been using `isValid` before and that was more readable.

Now Assert isInvalid is False. See the double negation? The tests now read completely opposite which makes me wonder if they're still even working as expected?